WordPress/src/main/res/values/blaze_styles.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Extracted from figma design
+Font family: SF Pro
+Line height: 22sp
+(identical to box height)
+translationY compensates for the line height adjustment of text
+-->
+    <!-- styles.xml -->
+    <style name="blaze_title">
+        <item name="android:textSize">17sp</item>
+        <item name="android:letterSpacing">-0.03</item>
+        <item name="android:textColor">?attr/wpColorOnSurfaceHigh</item>
+        <item name="android:lineSpacingExtra">2sp</item>
+    </style>
+
+    <style name="blaze_sub_title">
+        <item name="android:textSize">15sp</item>
+        <item name="android:letterSpacing">-0.02</item>
+        <item name="android:textColor">?attr/wpColorOnSurfaceMedium</item>
+<!--        <item name="android:textColor">#E6000000</item>-->
+        <item name="android:lineSpacingExtra">3sp</item>
```

Comment:
❓ Do we need a separate style file for Blaze styles?.

Reply:
Do we 100% need it? No.
Can we use this file for all blaze styles as the feature grows and it keep the clutter out of styles.xml? Yes. 

So it's a toss up and since I've already added it, I am going to leave it unless you have very strong feelings not to. 
 I will get rid of the commented line though. :)

Reply:
Hey @zwarm 

Going forward, we will rely more on Jetpack compose, which decreases the dependency on any XML file for Styles. So I think we can add this style to the styles.xml. 
There is no harm in having a separate file. Your call.